src/ProjectNeon/Assets/Data/Templates/PlayedCard.cs

```diff
@@ -2,16 +2,17 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class PlayedCard : ScriptableObject
+public class PlayedCard : MonoBehaviour
```

Comment:
`Data/Templates` folder is only for pure definition `ScriptableObject`

This should live in a feature folder in `Battle` section.